fv/health_test.go

```diff
@@ -227,9 +228,33 @@ var _ = Describe("health tests", func() {
 		felixLiveness = getHealthStatus(felixContainer.IP, "9099", "liveness")
 	}
 
+	Describe("healthHost not 'all interfaces'", func() {
+		// Since felix isn't running with hostnetworking, the host is checking for felix's readiness
+		// from a different networking namespace. We should not see that Felix is ready because that information
+		// is only accessible from inside the Felix container.
+		It("should run healthchecks on localhost by default", func() {
+			startFelix("", k8sInfra.GetDockerArgs, "", "", "")
+			Consistently(felixReady, "10s", "100ms").Should(Equal(statusErr))
+		})
+
+		It("should run support running healthchecks on '127.0.0.1'", func() {
+			startFelix("", k8sInfra.GetDockerArgs, "", "", "127.0.0.1")
+			Consistently(felixReady, "10s", "100ms").Should(Equal(statusErr))
+		})
+
+		It("should support running healthchecks on 'localhost'", func() {
+			startFelix("", k8sInfra.GetDockerArgs, "", "", "localhost")
+			Consistently(felixReady, "10s", "100ms").Should(Equal(statusErr))
+		})
+
```

Comment:
We don't have any dedicated infra, but checking from inside the container should be as simple as `felix.Exec("curl", "http://localhost:...")`.  I think that's worth doing assuming it's as easy as that.  WDYT?